internal/util/k8sutil/crd.go

```diff
@@ -60,3 +60,9 @@ func GetCRDManifestPaths(crdsDir string) (crdPaths []string, err error) {
 	})
 	return crdPaths, err
 }
+
+type CRDVersions []apiextv1beta1.CustomResourceDefinitionVersion
+
+func (vs CRDVersions) Len() int           { return len(vs) }
+func (vs CRDVersions) Less(i, j int) bool { return LessVersions(vs[i].Name, vs[j].Name) }
```

Comment:
Any reason to keep `LessVersions` around now? Maybe just inline `version.CompareKubeAwareVersionStrings(verA, verB) > 0`?

```suggestion
func (vs CRDVersions) Less(i, j int) bool { return version.CompareKubeAwareVersionStrings(vs[i].Name, vs[j].Name) > 0 }
```

Reply:
Yep, that works for me. The less code the better.